profiles/sftp_storage/README.md

```diff
@@ -0,0 +1,13 @@
+# sftp_storage
+
+## Usage
+
+*Add a description detailing what this profile does and any extra instructions for using it.*
+
```

Comment:
Remove the "instruction" comments from the readme.

Reply:
+1